nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -532,7 +532,9 @@ def do_one_task(self, client, num_of_threads, gpu, lock, timeout=60.0):
         )
         if gpu:
             command += " --gpu " + str(gpu)
-        _ = subprocess.Popen(shlex.split(command, True), preexec_fn=os.setsid, env=os.environ.copy())
+        new_env = os.environ.copy()
+        new_env["PYTHONPATH"] = ":".join(sys.path)[1:]
```

Comment:
Is this linux specific? Could it be done so different platforms are supported?

Reply:
Changed to use os.pathsep to work platform independent.

Reply:
Don't understand what problem this PR tries to solve.
Why should we explicitly add everything on sys.path to PYTHONPATH?

Reply:
Holger and one customer reported an issue running Simulator. 

When using sys.path.append or sys.path.insert doesn't seem to work with the simulator. I was able to reproduce the same issue also using just a python script, i.e. running python test.py. I cannot import the required modules inside the simulator run. However using export PYTHONPATH before running the script works fine. Interestingly, importing the module outside the SimulatorRunner works fine. The sys.path is not correctly passed on to subprocesses created by the simulator.